Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -212,14 +210,89 @@ angular.module('virtoCommerce.pricingModule')
+                } else {
+                    result.pricelistIds = blade.pricelistId ? [blade.pricelistId] : [];
+                }
+
                 return result;
             }
+            if (!$localStorage.exportSearchFilters) {
+                $localStorage.exportSearchFilters = [];
+            }
+
+            if (!$localStorage.exportSearchFilters[exportDataRequest.exportTypeName]) {
+                $localStorage.exportSearchFilters[exportDataRequest.exportTypeName] = [{ name: 'export.blades.export-generic-viewer.labels.new-filter' }];
+            }
+
+            $scope.exportSearchFilters = $localStorage.exportSearchFilters[exportDataRequest.exportTypeName];
+
+            if (!$localStorage.exportSearchFilterIds) {
+                $localStorage.exportSearchFilterIds = [];
+            }
+
+            $scope.exportSearchFilterId = $localStorage.exportSearchFilterIds[exportDataRequest.exportTypeName];
+
+            if ($scope.exportSearchFilterId) {
+                filter.current = _.findWhere($scope.exportSearchFilters, { id: $scope.exportSearchFilterId });
+            }
+
+            filter.change = function () {
+                $localStorage.exportSearchFilterId = filter.current ? filter.current.id : null;
+                var metafieldsId = exportDataRequest.exportTypeName + 'ExportFilter';
+                if (filter.current && !filter.current.id) {
+                    filter.current = null;
+                    showFilterDetailBlade({ isNew: true, metafieldsId: metafieldsId, exportTypeName: exportDataRequest.exportTypeName });
+                } else {
+                    bladeNavigationService.closeBlade({ id: 'exportGenericViewerFilter' });
+
+                    if (!filter.current) {
+                        $scope.resetRequestCustomFilter();
+                    }
+
+                    filter.criteriaChanged();
+                }
+            };
+
+            filter.edit = function () {
+                if (filter.current) {
+                    var metafieldsId = exportDataRequest.exportTypeName + 'ExportFilter';
+                    showFilterDetailBlade({ data: filter.current, metafieldsId: metafieldsId, exportTypeName: exportDataRequest.exportTypeName });
+                }
+            };
+
+            function showFilterDetailBlade(bladeData) {
+                var newBlade = {
+                    id: 'exportGenericViewerFilter',
+                    controller: 'virtoCommerce.exportModule.exportGenericViewerFilterController',
+                    template: 'Modules/$(VirtoCommerce.Export)/Scripts/blades/export-generic-viewer-filter.tpl.html'
+                };
+                angular.extend(newBlade, bladeData);
+                bladeNavigationService.showBlade(newBlade, blade);
+            }
+
+            filter.criteriaChanged = function () {
+                blade.refresh();
+            };
+
+            filter.resetKeyword = function () {
```

Comment:
Please check this commit: 
https://github.com/VirtoCommerce/vc-platform-core/commit/0073fc17ffa94183abffd9accbd0694591fae4e8

Here resetting custom filter is implemented. I think need to make the same here.

Reply:
Testing showed that there is no problem with filter clearance on this blade. So comment is not actual.